ClientKit/ClientKit.cs

```diff
@@ -30,6 +32,18 @@ namespace OSVR
 {
     namespace ClientKit
     {
+        public sealed class SafeClientContextHandle : SafeHandleZeroOrMinusOneIsInvalid
+        {
+            public SafeClientContextHandle() : base(true) { }
+
+            [ReliabilityContract(Consistency.WillNotCorruptState, Cer.MayFail)]
+            protected override bool ReleaseHandle()
+            {
+                System.Diagnostics.Debug.WriteLine("Client shutdown");

```

Comment:
Hmm, no automagic that I know of. I'll add "[OSVR]"  in front of these messages. Also, should this say "[OSVR] ClientContext shutdown" or just "[OSVR] Client shutdown"?